commerce.services.yml

```diff
@@ -34,6 +34,10 @@ services:
     tags:
       - { name: service_collector, tag: commerce.availability_checker, call: addChecker }
 
+  commerce.config_update:
```

Comment:
If the name of the class is ConfigUpdater, then the service name should match, commerce.config_updater